frontend/src/vendor/profile/VendorProfileInfo.js

```diff
@@ -2,6 +2,13 @@ import Products from "./products/Products";
 import VendorContacts from "./contacts/VendorContacts";
 import VendorIntroduction from "./VendorIntroduction";
 import Payment from "./payment/Payment";
+import styled from "styled-components";
+
+const ProfileComponents = styled.div`
+  height: var(--profile-components-height);
```

Comment:
Ez a változó szerintem eltűnt. Ha így is ugyanolyan jól néz ki, akkor nem is kellett. Ha nem, akkor javítsuk.